File: 15-word-freq/word_freq.py
# Python program to print the no of occurances of characters
# Created by M.V.Harish Kumar on 14-01-2024
s = input("Enter a string: ")
freq = {}
for c in s:
    if c not in freq:
        freq[c] = s.count(c)

print("The no of occurances are:")
for ch, count in freq.items():
    print(f"\t{ch!r}: {count}")

File: 15-word-freq/word_freq.php
<?php
// PHP program to print the no of occurances of characters
// Created by M.V.Harish Kumar on 18-09-2024
$s = readline("Enter a string: ");
$freq = [];
foreach (str_split($s) as $c) {
	if (!array_key_exists($c, $freq)) 
		$freq[$c] = 1;
	else
		$freq[$c]++;
	
}
echo "The no of occurances are:\n";
foreach ($freq as  $ch => $count) {
	echo "\t'$ch': $count\n";
}
